헤더, 옵션을 통한 검색 완료

skullkim opened the pull request.

skullkim committed:
헤더, 옵션을 통한 검색 완료

ndaemy commented on actProjects/App/static/js/search-play-with-options-cli.js:
```diff
@@ -0,0 +1,25 @@
+//'yyyy-mm-dd' 형식으로 날짜를 설정하기 위해 날짜 포맷 수정
+const correctDateFormat = (date) => {
+    return date < 10 ? '0' + date : date;
+}
```
혹시 파일 이름에 cli는 어떤 뜻인가요?

skullkim replied:
client라는 의미입니다.  아직 저희 팀의 컨벤션이 정해지지 않아서 제가 벡을 노드로 했을때 쓰던 규칙을 적용했습니다. 제가 노드로 할때는 파일 이름 자체에서 클라이언트용 js임을 알 수 있게 하기 위해 cli를 뒤에 붙입니다.

ndaemy commented on actProjects/App/static/css/search-play-with-options-style.css:
```diff
@@ -0,0 +1,69 @@
+
+.plays {
+    height: 30vh;
+    width: 15vw;
```
이런 식으로 width와 height가 화면 비례인 것은 포스터같이 이미지의 비율이 깨지면 안되는 곳에서는 좋지 않은 방법입니다. 고정 비율로 가주시는게 좋아요.

skullkim replied:
네 수정하겠습니다

ndaemy commented on actProjects/App/static/css/search-play-with-options-style.css:
```diff
@@ -0,0 +1,69 @@
+    height: auto;
+    /*margin-top: 3vh;*/
+    display: flex;
+    justify-content: space-evenly;
```
justify-content가 여기서 space-evenly이면, 한 row 에 5개 미만의 포스터가 올 경우 왼쪽 정렬이 되지 않고 가운데 정렬이 됩니다.
아직 정하진 않았지만 포스터같은 것들을 정렬할 때는 흔히 왼쪽정렬이므로, 요건 구현 방법을 조금 바꾸셔야 할 것 같아요

skullkim replied:
네 다른 방식으로 바꾸겠습니다

skullkim committed:
수정: 피드백 반영으로 인한 포스터 이미지 사이즈 정적으로 변환, 포스터 정렬 방식 변경

ndaemy commented:
코드리뷰 받으신것들 수정하시고 나면 Resolve conversation 해주시면 보기가 더 편하실거에요!

skullkim committed:
수정: 헤더 부분 css수정

skullkim committed:
수정: 검색 옵션 부분 css수정

skullkim committed:
옵션 검색 페이지에서 공연 이미지 로딩 완료

skullkim committed:
연극 프로필 사진 사이즈 수정

skullkim committed:
필요없는 코드 삭제, 주석 추가

Bhanjo commented:
search-play-with-options-style.css
header-style.css
혹시 vw, vh 단위들을 다른 단위로 바꿔주실 수 있나요?
브라우저창 확대 비율에 따라 스타일이 다 망가집니다ㅠ

ndaemy commented on actProjects/App/views.py:
```diff
@@ -11,11 +11,6 @@ def searchPage(request) :
-    #     img_data = base64.b64encode(img_file.read()).decode('utf-8')
-    # print(img_data)
-    # print(os.path.join(os.getcwd(), 'App/static/img'))
+    #이미 파일들의 이름을 읽어 온다.
```
`이미지 파일들` 이라고 쓰신거죠? ㅋㅋㅋㅋ

skullkim replied:
오타가 있었네요. 다음 커밋때 수정해서 올릴게요.

ndaemy commented on actProjects/App/static/css/header-style.css:
```diff
@@ -26,24 +26,34 @@ a {
 #header__search-play {
     display: flex;
     justify-content: center;
+    margin-top: 1vh;
```
한조님도 밑에 써주셨지만, margin 같은 것들은 vh로 쓰면 화면 사이즈 조정에 따라 깨질 위험이 높기 때문에 보통 고정 단위로 사용합니다

skullkim replied:
네, 수정하겠습니다.

ndaemy commented on actProjects/App/static/css/search-play-with-options-style.css:
```diff
@@ -13,14 +13,17 @@
+    padding: 1em;
+    background-clip: content-box;
+    box-shadow: inset 0 0 0 0.5em lightgrey;
+    /* background-color: lightgrey; */
```
1. padding-bottom과 padding이 중복해서 들어가면 의미가 없습니다.
2. background가 없다면 background-clip의 의미가 없는데, 혹시 추후에 background를 넣을 예정이신건가요?

skullkim replied:
1. 이거는 수정하도록 하겠습니다.
2. background를 넣을 예정은 없습니다. padding에만 색을 넣기 위해 저렇게 한거입니다.

skullkim committed:
옵션 검색이 되는 것처럼 구현 완료, 수정: views.py에 주석 오타 수정

skullkim committed:
옵션 검색 페이지 css 단위 수정

skullkim committed:
header css 단위 수정

skullkim committed:
수정:포스터 랜덤으로 보여지는 부분 수정
시연용 포스터 하나 맨 앞에 고정

skullkim committed:
수정:헤더 부분에서 My page 이동 링크 삭제
현재는 프로토타입 구현이고 MVP에 회원정보 저장이 포함되있지 않다.
따라서 로그인을 해야 들어갈 수 있는 My page 링크를 삭제했습니다.

skullkim committed:
Merge branch 'master' into yunkiKim
